.eslintrc.js

```diff
@@ -31,8 +31,38 @@ module.exports = {
         {
             files: ['*.ts', '*.tsx'],
             rules: {
+                'flowtype/no-types-missing-file-annotation': 'off',
```

Comment:
✅ this is ok from the AFT side - we will need to take a look at the gaps in the shared config and address them